PlottingAnalysis/TS_TopFeatures.m

```diff
@@ -125,10 +147,12 @@
     classNumbers = arrayfun(@(x)sum(TimeSeries.Group==x),1:numClasses);
     isBalanced = all(classNumbers==classNumbers(1));
     if isBalanced
-        fn_testStat = GiveMeFunctionHandle(whatTestStat,numClasses,'acc',0);
+        fn_testStat = @(XTrain,yTrain,XTest,yTest,numFolds) GiveMeCfn(whatTestStat,...
```

Comment:
The whole point of `GiveMeFunctionHandle` is that it is more efficient when you use `fast_linear` or have a binary problem (and it outsources to the slower more general model from `GiveMeCfn` only when it has to)… What was the reasoning for replacing it with an inline function here?

Reply:
I've mostly replaced `GiveMeCfn` so that we don't really need `GiveMeFunctionHandle` any more (I think). Check over it and see if we can just bypass `GiveMeFunctionHandle` with my latest changes